### ISSUE TITLE
MQTT doesn't report temperature sensors unless at least 1 IO is enabled

### ISSUE THREAD

**Issue by nielsonm236** [nielsonm236/NetMod-ServerApp#66]
Reported by jmcvieira1: Disable all i/o port´s leave mqtt and DS18B20 turned on save and reboot, and the module will not connect to the mqtt server. I have to enable one port to the module connect to the server again.

**Comment by nielsonm236** [comment]
I found a check in code where I was requiring at least one I/O pin be enabled for MQTT to connect. I'm not sure why that check was originally put in the code other than the boot process is quicker if there are no resources enabled ... but I've removed the check and now MQTT will connect even if no I/O pins are enabled. Now if the only thing enabled is the DS18B20 sensors MQTT will connect properly and the temperature sensors display properly in HA.

FYI, I rebooted with all I/O disabled and the DS18B20 checkbox "off". The MQTT connect process completes as it should even though there are no resources enabled on the Network Module.

This fix will be applied in the next release.
